cluster-autoscaler/core/scale_down.go

```diff
@@ -648,6 +666,18 @@ func (sd *ScaleDown) updateUnremovableNodes(nodes []*apiv1.Node) {
 	}
 }
 
+func (sd *ScaleDown) clearUnremovableNodeReasons() {
+	sd.unremovableNodeReasons = make(map[string]*simulator.UnremovableNode)
+}
+
+func (sd *ScaleDown) addUnremovableNodeReasonSimple(node *apiv1.Node, reason simulator.UnremovableReason) {
+	sd.unremovableNodeReasons[node.Name] = &simulator.UnremovableNode{Node: node, Reason: reason, BlockingPod: nil}
+}
+
+func (sd *ScaleDown) addUnremovableNodeReasonFull(unremovableNode *simulator.UnremovableNode) {
```

Comment:
nit: consider renaming to 'addUnremovableNode'

Reply:
Done